File: 0137_Single Number II/SingleNumberII.java
public class Solution {
    public int singleNumber(int[] A) {
        int ans = 0;
        for(int i=0;i<32;i++){
            int bit = 0;
            for(int j=0;j<A.length;j++){
                bit += ((A[j]>>i) & 1);
            }            
            ans += ((bit % 3) << i );
        }        
        return ans;
    }
}

File: 0137_Single Number II/SingleNumberII.cs
public class Solution {
    public int SingleNumber(int[] nums)
    {
        var ans = 0;

        for (int i = 0; i < 32; i++)
        {
            var count1 = 0;
            foreach (var n in nums)
            {
                count1 += (n >> i) & 1;
            }

            ans += ((count1 % 3) << i);
        }

        return ans;
    }
}